<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="User" name="testRunner.TestRun" tests="1" failures="0" timestamp="17 Sep 2020 18:19:07 GMT" time="16.329" errors="1">
  <testcase name="runScenario" time="16.329" classname="testRunner.TestRun">
    <error type="io.cucumber.core.runner.AmbiguousStepDefinitionsException" message="&quot;User opens URL &quot;https://ijmeet.com/&quot;&quot; matches more than one step definition:
  &quot;User opens URL {string}&quot; in stepDefination.SignInSteps.user_opens_url(java.lang.String)
  &quot;^User opens URL &quot;([^&quot;]*)&quot;$&quot; in stepDefination.HostMeetngStep.user_opens_url_something(java.lang.String)">
      <![CDATA[io.cucumber.core.runner.AmbiguousStepDefinitionsException: "User opens URL "https://ijmeet.com/"" matches more than one step definition:
  "User opens URL {string}" in stepDefination.SignInSteps.user_opens_url(java.lang.String)
  "^User opens URL "([^"]*)"$" in stepDefination.HostMeetngStep.user_opens_url_something(java.lang.String)
at io.cucumber.core.runner.CachingGlue.findStepDefinitionMatch(CachingGlue.java:371)
at io.cucumber.core.runner.CachingGlue.stepDefinitionMatch(CachingGlue.java:339)
at io.cucumber.core.runner.Runner.matchStepToStepDefinition(Runner.java:146)
at io.cucumber.core.runner.Runner.createTestStepsForPickleSteps(Runner.java:126)
at io.cucumber.core.runner.Runner.createTestCaseForPickle(Runner.java:109)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:70)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$0(TestNGCucumberRunner.java:115)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:117)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:112)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRunner.TestRun -->
